resources/linux/rpm/code.spec.template

```diff
@@ -4,17 +4,17 @@ Release:  @@RELEASE@@.el7
 Summary:  Code editing. Redefined.
 Group:    Development/Tools
 Vendor:   Microsoft Corporation
-Packager: Visual Studio Code Team <vscode-linux@microsoft.com>
+Packager: Azure Data Studio Team
 License:  @@LICENSE@@
-URL:      https://code.visualstudio.com/
+URL:      https://aka.ms/azuredatastudio
 Icon:     @@NAME@@.xpm
 Requires: @@DEPENDENCIES@@
 AutoReq:  0
 
 %global __provides_exclude_from ^%{_datadir}/%{name}/.*\\.so.*$
 
 %description
-Visual Studio Code is a new choice of tool that combines the simplicity of a code editor with what developers need for the core edit-build-debug cycle. See https://code.visualstudio.com/docs/setup/linux for installation instructions and FAQ.
```

Comment:
This and the summary don't really make sense for ADS either, so we should probably update those if we're going to fix the names.